### DISCUSSION TITLE
Dispatch workflow sample JSON

### DISCUSSION THREAD

**Discussion by rmanjusha** [elsa-workflows/elsa-core#5339] [Q&A]
Please provide Dispatch workflow sample JSON

**Comment by glime-ai** [comment]
To provide you with a sample JSON for dispatching a workflow in Elsa, I can refer you to the examples provided in the context. However, the context does not include a complete JSON representation of a workflow that includes the `DispatchWorkflow` activity. Instead, it describes how to set up and use the `DispatchWorkflow` activity within Elsa Studio and provides downloadable examples.

If you are looking for a JSON example to create or modify workflows programmatically, you would typically need to look at the JSON export of a workflow that has been designed in Elsa Studio with the `DispatchWorkflow` activity configured.

Since the context does not provide a direct JSON example, I recommend downloading the example workflows provided in the context and then exporting them to JSON from Elsa Studio. This way, you can see the exact structure of the JSON for a workflow that includes the `DispatchWorkflow` activity.

To try out the example and obtain the JSON, you can download the workflows from the following links and import them into Elsa Studio:

- For dispatching a workflow with no input and no output: [dispatching-child-workflows-demo1.zip](https://v3.elsaworkflows.io/guides/dispatching-child-workflows/dispatching-child-workflows-demo1.zip)
- For dispatching a workflow with input: [dispatching-child-workflows-demo2.zip](https://v3.elsaworkflows.io/guides/dispatching-child-workflows/dispatching-child-workflows-demo2.zip)
- For dispatching a workflow and receiving output: [dispatching-child-workflows-demo3.zip](https://v3.elsaworkflows.io/guides/dispatching-child-workflows/dispatching-child-workflows-demo3.zip)

After importing these workflows into Elsa Studio, you can export them to JSON to see the structure.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=dRbvTb7IhCSYiI&utm_medium=github).
